### DISCUSSION TITLE
git pull error: The following untracked working tree files would be overwritten by merge

### DISCUSSION THREAD

**Discussion by Naveen23kumar** [community/community#62958] [Repositories] [answered]
### Select Topic Area

Question

### Body

ERROR

**Comment by mvkaran** [comment]
@Naveen23kumar 👋🏼 

I am not sure of what's the complete error, but from the looks of it, it seems like you have some files within your local git repository that haven't been version-controlled (i.e, not committed to git before). When the upstream git repository has those files, but not your local repository, git pull would usually fail with that error.

Your option would be to either remove those untracked files by deleting them, or by committing them to the git repo and then pulling in changes from upstream by using `git pull`, and then rebasing if needed.

**Comment by mehulrajdev** [comment, marked as answer]
Your option would be to either remove those untracked files by deleting them, or by committing them to the git repo and then pulling in changes from upstream by using git pull, and then rebasing if needed.
